src/app/pages/dynamic-stepper/dynamic-stepper.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DynamicStepperComponent } from './dynamic-stepper.component';
+import { DynamicStepperModule } from './dynamic-stepper.module';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatRadioModule } from '@angular/material/radio';
+import { MatStepperModule } from '@angular/material/stepper';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+describe('DynamicStepperComponent', () => {
+    let component: DynamicStepperComponent;
+    let fixture: ComponentFixture<DynamicStepperComponent>;
+
+    beforeEach(async(() => {
+        void TestBed.configureTestingModule({
+            imports: [
+                DynamicStepperModule,
```

Comment:
You should be able to replace all these individual imports with DynamicStepperModule